great_expectations/experimental/datasources/spark_datasource.py

```diff
@@ -11,11 +11,11 @@
     _FilesystemDataAsset,
 )
 from great_expectations.experimental.datasources.interfaces import (
+    BatchSorter,
```

Comment:
@NathanFarmer This module is being renamed in #7197 (to indicate that it is filesystem-only).  Thanks.